### PR TITLE
bg_ikana_shutter OK

### PR DESCRIPTION
**Pull request by kyleburnette**
Completely decomped, but I'd like to work on some function names before merging

### CODE REVIEW

**Review by Kenix3 — COMMENTED** [review]
Code looks good to me.
Will wait to review names once you get everything in at once.
Also pending build test.
Were you meaning to update the submodules tools/asm-duffer and tools/asm-processor?

File: src/overlays/actors/ovl_Bg_Ikana_Shutter/z_bg_ikana_shutter.c [hunk 1]
```diff
@@ -21,50 +39,173 @@ const ActorInit Bg_Ikana_Shutter_InitVars = {
...
+
+void func_80BD5A18(BgIkanaShutter* this, GlobalContext* globalCtx) {
+    s32 pad[2];
+    s16 quake;
+
+    this->dyna.actor.velocity.y += -5.0f;
+    this->dyna.actor.velocity.y *= 0.978f;
+    this->dyna.actor.world.pos.y += this->dyna.actor.velocity.y;
+    if (this->dyna.actor.world.pos.y <= this->dyna.actor.home.pos.y) {
+        quake = Quake_Add(ACTIVE_CAM, 3U);
```
Comment (Rozelette) [inline src/overlays/actors/ovl_Bg_Ikana_Shutter/z_bg_ikana_shutter.c:149]: Minor nit
```suggestion
        quake = Quake_Add(ACTIVE_CAM, 3);
```

File: src/overlays/actors/ovl_Bg_Ikana_Shutter/z_bg_ikana_shutter.c [hunk 2]
```diff
@@ -21,50 +39,173 @@ const ActorInit Bg_Ikana_Shutter_InitVars = {
...
+        Quake_SetSpeed(quake, 0x5420);
+        Quake_SetQuakeValues(quake, 4, 0, 0, 0);
+        Quake_SetCountdown(quake, 0xC);
```
Comment (Rozelette) [inline src/overlays/actors/ovl_Bg_Ikana_Shutter/z_bg_ikana_shutter.c:152]:
```suggestion
        Quake_SetCountdown(quake, 12);
```

**Review by Rozelette — COMMENTED** [review]
LGTM
I'll hold off on approving until you work on the names like you mentioned.

**Review by Rozelette — APPROVED** [review]
LGTM

**Pull request merged**